source/libraries/CompileSolidity.ts

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env node
 
 import * as fs from 'fs';
+import * as readFile from 'fs-readfile-promise';
 import * as mkdirp from 'mkdirp';
 import * as path from 'path';
 import * as recursiveReadDir from 'recursive-readdir';
-import { CompilerInput, CompilerInputSourceCode, CompilerInputSourceFile, compileStandardWrapper } from 'solc';
+import { CompilerInput, CompilerInputSourceCode, CompilerInputSourceFile, CompilerOutput, CompilerOutputError, CompilerOutputContractAbi, CompilerOutputEvmBytecode, compileStandardWrapper } from 'solc';
```

Comment:
You only need to import classes/interfaces that you will be using directly.  In this case, I think only `CompilerOutput` is referenced so that is the only thing you need to import.

Reply:
Good to know.  I only included them because we were already including CompilerInputSourceCode and CompilerInputSourceFile, which aren't directly referenced anywhere.  I'll remove all the unused ones.